src/Hazelcast.Net/DistributedObjects/IHMap.Setting.cs

```diff
@@ -265,6 +265,6 @@ public partial interface IHMap<TKey, TValue> // Setting
         /// Time resolution for <paramref name="timeToLive"></paramref> is seconds. The given value is rounded to the next closest second value.
         /// </para>
         /// </remarks>
```

Comment:
Javadoc sync from Java missing: exception throws missing.
server config if negative, can be different from -1. can you double check with javadoc?